DUI3-DX/Converters/Rhino/Speckle.Converters.Rhino7/ToSpeckle/Raw/PolylineToSpeckleConverter.cs

```diff
@@ -32,12 +35,12 @@ public class PolylineToSpeckleConverter
   /// <param name="target">The Rhino polyline to be converted.</param>
   /// <returns>The converted Speckle polyline.</returns>
   /// <remarks>⚠️ This conversion assumes domain interval is (0,LENGTH) as Rhino Polylines have no domain. If needed, you may want to use PolylineCurve conversion instead. </remarks>
-  public SOG.Polyline Convert(RG.Polyline target)
+  public SOG.Polyline Convert(IRhinoPolyline target)
   {
-    var box = _boxConverter.Convert(new RG.Box(target.BoundingBox));
+    var box = _boxConverter.Convert(_rhinoBoxFactory.CreateBox(target.BoundingBox));
     var points = target.Select(pt => _pointConverter.Convert(pt)).ToList();
 
-    if (target.IsClosed)
+    if (points[0] == points[^1] && target.IsClosed)
```

Comment:
Why was this added?

Reply:
This was done with @clairekuang about a Rhino issue for V3 that isn't in V2